src/accountsdb/swiss_map.zig

```diff
@@ -624,22 +624,22 @@ fn generateData(allocator: std.mem.Allocator, n_accounts: usize) !struct {
 
 pub const BenchmarkSwissMap = struct {
     pub const min_iterations = 1;
-    pub const max_iterations = 100;
+    pub const max_iterations = 1_000;
 
     pub const BenchArgs = struct {
         n_accounts: usize,
         name: []const u8 = "",
     };
 
     pub const args = [_]BenchArgs{
-        BenchArgs{
-            .n_accounts = 100_000,
-            .name = "100k accounts",
-        },
-        BenchArgs{
-            .n_accounts = 500_000,
-            .name = "500k accounts",
-        },
+        // BenchArgs{
```

Comment:
Can it be removed instead? if not, can there be a comment explaining why it is commented out?

Reply:
09842663